demo-shell-ng2/app/components/files/files.component.html

```diff
@@ -48,29 +45,13 @@
                 </md-menu>
             </adf-toolbar>
         </ng-container>
-        <adf-document-list
-                #documentList
-                [permissionsStyle]="permissionsStyle"
-                [creationMenuActions]="!useCustomToolbar"
-                [currentFolderId]="currentFolderId"
-                [contextMenuActions]="true"
-                [contentActions]="true"
-                [allowDropFiles]="true"
-                (error)="onNavigationError($event)"
-                (success)="resetError()"
-                (preview)="showFile($event)"
-                (permissionError)="handlePermissionError($event)">
+        <adf-document-list #documentList [permissionsStyle]="permissionsStyle" [creationMenuActions]="!useCustomToolbar" [currentFolderId]="currentFolderId"
```

Comment:
Can we get back the multiline approach for the attributes to have better readability?